grants_tagger/__main__.py

```diff
@@ -362,6 +363,10 @@ def explain():
     # feature importance for models
     pass
 
+@app.command()
+def visualize():
+    st_app_path = os.path.join(os.path.dirname(__file__), "streamlit_visualize.py")
+    subprocess.Popen(["streamlit", "run", st_app_path])
```

Comment:
Wonder if there is a way to run Streamlit programatically, like Flasks' `import app; app.run()`

Reply:
I did not find a way, but will look again. Btw @jdu is this a good way if invoking a script? I remember hunter had mentioned that some python way was better than another

Reply:
There's a few different ways of spawning a sub-process across the different python versions. This is probably fine for what is just a labs component.